addons/account_facturx/data/facturx_templates.xml

```diff
@@ -231,5 +231,86 @@
                 </rsm:SupplyChainTradeTransaction>
             </rsm:CrossIndustryInvoice>
         </template>
+
+        <template id="account_invoice_pdfa_3_facturx_metadata">
+            <x:xmpmeta xmlns:x="adobe:ns:meta/">
+                <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+                    <rdf:Description xmlns:pdfaid="http://www.aiim.org/pdfa/ns/id/" rdf:about="">
```

Comment:
Is empty `rdf:about` really needed?

Reply:
It is written as such in the example given in the factur-x doc/zip

I didn't try without it but I kept it like in the example to be sure